nixos/modules/services/networking/searx.nix

```diff
@@ -66,6 +64,9 @@ in
         serviceConfig = {
           User = "searx";
           ExecStart = "${cfg.package}/bin/searx-run";
+          StateDirectory = "searx";
+          # Allow running the service on priviledged ports.
+          AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
```

Comment:
I'm not sure if we want to raise that capability for everyone using searx (AFAICS we don't do it that often for web-services atm).

If you prefer that kind of solution you can still modify the service in your `configuration.nix`:

```nix
{
  services.searx.enable = true;
  systemd.services.searx.serviceConfig.AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
}
```

Reply:
Ok, got it... I managed to use searx as a proxy with apache, so I don't need a priviledged port either.
  
```
  # Startup a searx server.
  services.searx.enable = true;
  services.searx.configFile = "/etc/nixos/searx-config.yml";

 # Apache httpd server configuration.
  services.httpd.enable = true;
  services.httpd.package = pkgs.apacheHttpd_2_4;
  services.httpd.extraModules = [ "headers" "proxy" "proxy_http" ];
  services.httpd.virtualHosts."searx.example.com" = {
    extraConfig = ''
      ProxyPass "/" "http://192.168.100.101:8100/"
      ProxyPassReverse "/" "http://192.168.100.101:8100/"
    '';
  };
```